src/main/java/frc/robot/Robot.java

```diff
@@ -24,6 +28,18 @@
   private String m_autoSelected;
   private final SendableChooser<String> m_chooser = new SendableChooser<>();
 
+  // Talons and Victors
+  WPI_TalonSRX rightTalon = new WPI_TalonSRX(6);   
+  WPI_TalonSRX leftTalon = new WPI_TalonSRX(5); 
+  WPI_VictorSPX rightVictor = new WPI_VictorSPX(1);
+  WPI_VictorSPX leftVictor = new WPI_VictorSPX(4);
```

Comment:
There should be a comment here as to how the ports here work. As I recall, these may be mapped in software. For now, a TODO for documenting these later into the season when the electronics are further along should suffice.

Reply:
A TODO has been commented for now.